File: modul 3/python/PRAK304-2410817110008-MuhammadAzmaAlFaqih.py
a = int(input())
b = int(a/10)
if a == 0:
    print("Nol")
elif b == 0:
    print("Satuan")
elif b == 1:
    print("Belasan")
elif b >= 10:
    print("Anda Menginput Melebihi Limit Bilangan")
else:
    print("Puluhan")

File: modul 3/c/PRAK304-2410817110008-MuhammadAzmaAlFaqih.c
#include <stdio.h>

int main(){
  int a;
  scanf("%d", &a);

  if (a==0){
    printf("Nol");
  } else if (a/10 == 0){
    printf("Satuan");
  } else if (a/10 == 1){
    printf("Belasan");
  } else if (a/10 > 1 && a/10 < 10){
    printf("Puluhan");
  } else {
    printf("Anda Menginput Melebihi Limit Bilangan");
  }
  printf("\n");
}
